develocity-maven.yml

```
@@ -41,6 +41,16 @@ spec:
   function createTmp() {
     export TMP_GE=$(mktemp -d ge.XXXXXX --tmpdir="${CI_PROJECT_DIR}")
   }
+  
+  function isAtLeast() {
+    local v1=$(printf "%03d%03d%03d%03d" $(echo "$1" | cut -f1 -d"-" | tr '.' ' '));
+    local v2=$(printf "%03d%03d%03d%03d" $(echo "$2" | cut -f1 -d"-" | tr '.' ' '));
+    if [ "$v1" -eq "$v2" ] || [ "$v1" -gt "$v2" ]; then
+      echo "true";
+    else
+      echo "false";
+    fi
+  }
```

You can get rid of the `;`, see:
```suggestion
  function isAtLeast() {
    local v1=$(printf "%03d%03d%03d%03d" $(echo "$1" | cut -f1 -d"-" | tr '.' ' '))
    local v2=$(printf "%03d%03d%03d%03d" $(echo "$2" | cut -f1 -d"-" | tr '.' ' '))
    if [ "$v1" -eq "$v2" ] || [ "$v1" -gt "$v2" ]
    then
      echo "true"
    else
      echo "false"
    fi
  }
```

---

Fixed in https://github.com/gradle/develocity-gitlab-templates/pull/22/commits/e52dfa902cf9a3128ad25cc95fb835ad68ad8876